victor/src/layout/boxes/mod.rs

```diff
@@ -16,7 +16,9 @@ pub(super) enum FormattingContext {
 }
 
 #[derive(Debug)]
-pub(super) struct BlockFormattingContext(pub BlockContainer);
+pub(super) struct BlockFormattingContext {
+    pub container: BlockContainer,
```

Comment:
While this field is a (block) container, in relation to the BFC it is the BFC’s contents. What do you think of naming the field `contents`?